parent/pom.xml

```diff
@@ -1268,6 +1274,17 @@
       </pluginManagement>
       <plugins>
          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <extensions>true</extensions>
+            <configuration>
+               <!-- See configuration details at http://books.sonatype.com/nexus-book/reference/staging-deployment.html -->
+               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
+               <serverId>${jboss.releases.repo.id}</serverId>
+               <autoReleaseAfterClose>true</autoReleaseAfterClose>
```

Comment:
with this setting, will it do both of:
 - close the repository
 - release it
Or only one of these steps?